pkg/detectors/dynadot/dynadot.go

```diff
@@ -0,0 +1,78 @@
+package dynadot
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+	"io/ioutil"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"dynadot"}) + `\b([a-z0-9A-Z]{30})`)
```

Comment:
suggestion: this regex can be made stricter by closing the word boundary at the end `\b([a-z0-9A-Z]{30})\b` so the keys will strictly matches the length provided in the regex.